### ISSUE TITLE
Hint for htaccess option is missing some text and contains invalid link

### ISSUE THREAD

**Issue by andreimilto** [terreng/simple-web-server#149]
*Simple Web Server v1.2.8 win portable*

<img width="960" alt="swsbug" src="https://github.com/terreng/simple-web-server/assets/61744578/c0f9754d-d577-467d-96f7-c3c9bff82c5e">

In the UI, in server settings there's an option `Enable .swshtaccess configuration files`. Clicking on a question mark icon next to it opens up a dialog window with description text. Part of the text from the following line is missing in the dialog:

https://github.com/terreng/simple-web-server/blob/ce0c9bdf0c3b09eddc9470d7dc3367bc8f496fcd/lang.js#L97

Instead the missing text fragment has gotten into the URL of the anchor `plugins` making it invalid:
```
https://simplewebserver.org/docs/swsaccess.html'>Advanced configuration using .swshtaccess files</a>.<br><br>This functionality is experimental and may change in future releases. Consider using <a href='https://simplewebserver.org/docs/plugins.html
```

**Comment by terreng** [comment]
Thanks so much for catching this bug! We really appreciate your recent contributions.

**Comment by terreng** [comment]
Landed in 1.2.9.
